platform/viewer/src/utils/initWebWorkers.js

```diff
@@ -0,0 +1,22 @@
+import cornerstoneWADOImageLoader from 'cornerstone-wado-image-loader';
+
+let initialized = false;
+
+export default function initWebWorkers() {
+  const config = {
```

Comment:
I might be missing something, but why do we have 3 separate `initiwebWorkers` for CornerstoneWADOImageLoader (two of them are new)

- here
- extensions/cornerstone/src/init.js
- platform/viewer/src/config.js